indico/MaKaC/common/cache.py

```diff
@@ -481,6 +514,7 @@ def __repr__(self):
         return 'GenericCache(%r)' % self._namespace
 
     def _connect(self):
+        """Connect to the CacheClient. Must be called before accessing ``self._client``."""
```

Comment:
```
"""Connect to the CacheClient.

This must be called before accessing ``self._client``.
"""
```

(first line with a "short" description, then details in the following lines after an empty line, similar to what we do for git commits)